specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "description": "Specifies the properties of the private endpoint connection."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PrivateEndpointConnectionListResult": {
+      "title": "PrivateEndpointConnections",
+      "description": "List of private endpoint connections.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "title": "PrivateEndpointConnection",
+          "description": "Specifies the name of the private endpoint connection.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        }
+      }
+    },
+    "PrivateEndpointConnectionProperties": {
+      "title": "PrivateEndpointConnectionProperties",
+      "type": "object",
+      "properties": {
+        "privateEndpoint": {
+          "$ref": "#/definitions/PrivateEndpoint",
+          "description": "Specifies the private endpoint."
+        },
+        "privateLinkServiceConnectionState": {
+          "$ref": "#/definitions/ConnectionState",
+          "description": "Specifies the connection state."
+        }
+      }
+    },
+    "ActionsRequired": {
+      "title": "ActionsRequired",
+      "type": "string",
+      "enum": [
+        "None",
+        "Recreate"
+      ]
+    },
+    "PrivateEndpoint": {
+      "title": "PrivateEndpoint",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Specifies the id of private endpoint."
+        }
+      }
+    },
+    "PrivateLinkResourcesListResult": {
+      "title": "PrivateLinkResourcesListResult",
+      "description": "Specifies the private link resource.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of private endpoint connection resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        }
+      }
+    },
+    "PrivateLinkResource": {
+      "description": "A private link resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "description": "Fully qualified identifier of the resource.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the resource.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkResourceProperties": {
```

Comment:
>PrivateLinkResourceProperties [](http://example.com/codeflow?start=5&length=29)

Consider using type from common-types.

Reply:
will this property as the resource emits more information than in common-type.